### DISCUSSION TITLE
Use more accurate terms in docs or help.

### DISCUSSION THREAD

**Discussion by 6lj6** [helix-editor/helix#6673] [General]
h
1. is  not `Move left`
2. is `Move logical cursor backward(direction) of text by a step of char if possible`, it contains 4 different subclasses
    1. if logicalLine is shorter than screenLine => `move cursor left per char in line`
    2. if at first end of two ends of linear logical chars(text), => `nothing`
    3. if at first end of two ends of linear screenline chars(text) => `move cursor up to the end of above line` where meets the break char `\n` (ENTER)
    4. if logicalLine is longer than screenLine, it won't move visualScreenCusor, but moving LogicalLine right, therefore logicalCusor move to left on logical chars.

j
1. is not `Move up`
2. is `Move between \n and stay the nearest position of chars of original line except other operation`
    1. So it change the position of the row's index
```
b
    backward cursor to a wordStart(between \n SPACE), select a word of that wordStart
        select with space at right if have
            without \n
        TAB \t counts a selection
        punctuation counts as a selection
        dual punctuation () counts as one selection
        N b will only select 1 word instead of N words
```

**Comment by the-mikedavis** [comment]
Documenting the nuances of the motions could fit in https://github.com/helix-editor/helix/pull/997

This is too verbose to fit in the current command documentation though.

**Reply by 6lj6** [reply]
If the degree of abstraction in doc need to be high(concise), is it possible to add a low-level(detailed) specification :smiley:
